### ISSUE TITLE
getCacheBusterParam doesn't work as documented

### ISSUE THREAD

**Issue by lookbusy1344** [bigskysoftware/htmx#2368]
Regarding `htmx.config.getCacheBusterParam` the documentation says

“defaults to false, if set to true htmx will include a cache-busting parameter in GET requests to avoid caching partial responses by the browser”

But the only effect of setting `true` seems to appending a static id to the request:

```
https://localhost:7203/?handler=MyHandler&org.htmx.cache-buster=myDiv
```

Digging in the source I found:

https://github.com/bigskysoftware/htmx/blob/master/src/htmx.js#L3203

```
if (htmx.config.getCacheBusterParam && verb === 'get') {
   filteredParameters['org.htmx.cache-buster'] = getRawAttribute(target, "id") || "true";
}
```

Introduced entirely in 613f7b61d59e8ac4741e02d9f468ca635773ab2d (9 Jan 2023), its not been modified since.

The documented behaviour is what I’d like, but it seems different from what’s implemented.

Is this a bug? Is this a fix:

```
if (htmx.config.getCacheBusterParam && verb === 'get') {
   filteredParameters['org.htmx.cache-buster'] = new Date().getTime();
}
```

**Comment by lookbusy1344** [comment]
There’s some debate as to whether I am misinterpreting the term “cache-busting”. Every reference I find says, to quote Google,

“Cache busting is a way to prevent browsers from caching your content. Cache busting typically adds a random string to your request — typically a random number or timestamp”

Perhaps if the observed behaviour is intentional the docs should be altered to say

“defaults to false, if set to true htmx will include the hx-target ID in the request, allowing caching by target. Eg.. “ etc

**Comment by lookbusy1344** [comment]
Fixed in f9591c9790ee970f8fac46851d570e1d8fdb19ab
